Documentation/markdown/ReleaseNotes.md

```diff
@@ -37,6 +37,8 @@ This work has been published with [JOSS](https://github.com/openjournals/joss-re
 
   * Supporting IfcClothoid ([#482](https://github.com/tumcms/Open-Infra-Platform/pull/482))
 
+  * Supporting IfSine ([#553](https://github.com/tumcms/Open-Infra-Platform/pull/553))
+
```

Comment:
Please move this to the work in progress section.
This section here (4.0.0) is the version when the JOSS paper was published.
From then on new additions will be part of the next OIP version (work in progress).

Reply:
Resolved in [930a993](https://github.com/tumcms/Open-Infra-Platform/pull/553/commits/930a993c674f83c2ea867f44691d9772f039f221)